docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -27,15 +27,21 @@ OpenProject offers close integration with Nextcloud to allow users to:
 
 > Note: This guide only covers the integration setup. Please go to our [Nextcloud integration user guide](../../../user-guide/file-management/nextcloud-integration/) to learn more about how to work with the Nextcloud integration.
 
-## Minimum requirements
+## Required system versions
 
-Please note these minimum version requirements for the integration to work with a minimal feature set:
+OpenProject makes significant efforts to ensure that the integration works with all the latest stable versions.  We strongly recommend continuously updating all systems to the current stable version. This ensures access to the latest features and prompt fixes for any potential bugs.
 
-- OpenProject version 12.2 (or above)
-- Nextcloud version 22 (or above)
-- The [OpenProject Integration](https://apps.nextcloud.com/apps/integration_openproject) app, version 2.0.0 (or above)
+- OpenProject [latest release](https://www.openproject.org/docs/release-notes/).
+- The latest version of the [OpenProject Integration Nextcloud app](https://apps.nextcloud.com/apps/integration_openproject)  from the Nextcloud App Store.
+- Nextcloud version in the latest `stable` version. 
+  - If you run Nextcloud in the community edition be careful to not 
+    blindly follow the update hints in the administration area of a Nextcloud instance, as they nudge you to use the
+    `latest` version, which might not be the latest `stable` version. We recommend to check the [tags of Nextcloud's official
+    Docker image](https://hub.docker.com/_/nextcloud). There you can see which version is currently the stable one. 
+  - If you installed Nextcloud via Docker then we advice you to pin it to a `stable` tag.
 
-We recommend using the latest versions of both Nextcloud and OpenProject to be able to use the latest features. 
+
+We recommend using the latest stable versions of both Nextcloud and OpenProject to be able to use the latest features. 
```

Comment:
@MayaBerd I removed that line on purpose as that is a duplication of the text above the bullet list.

Reply:
Will you remove it again?

Reply:
@wielinde yes, just did